pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -6541,7 +6430,7 @@ spec:
 		map[string]string{},
 	)
 
-	if d := cmp.Diff(actualTaskRun, expectedTaskRun); d != "" {
+	if d := cmp.Diff(expectedTaskRun, actualTaskRun); d != "" {
```

Comment:
Thank you for fixing these changes! would you be OK splitting changes to pipelinerun tests that aren't related to this PR into a separate one?

Reply:
These changes are so minor that I'd prefer to just leave them in here if that's alright.